src/de/jost_net/JVerein/gui/action/MitgliedArbeitseinsatzZuordnungAction.java

```diff
@@ -85,7 +86,10 @@ else if (context instanceof Mitglied[])
       }
       Logger.error("Fehler", e);
     }
-
+    catch (OperationCanceledException oce)
+    {
+      throw oce;
+    }
```

Comment:
Wenn man den Dialog mit mit Close schliest (x rechts oben) dann ist part == null und es kommt zu einer Exception. Man sollte in Zeile 60 abprüfen ob part == 0 und dann nichts tun.

Reply:
habe ich gemacht